README.md

```diff
@@ -520,3 +520,17 @@ Support for sending-side autoscaling:
 - [RabbitMQ](https://www.rabbitmq.com/)
 - [ActiveSupport::Notification](http://api.rubyonrails.org/classes/ActiveSupport/Notifications.html)
 - [Pusher](https://app.pusher.com/)
+
+## Docker
+
+This project contains a `Dockerfile` and a Docker image is being built on every CI run to ensure smoother transition to a Docker-based architecture. Normally that step would not require any manual input from you as a developer but you may still want to manually check if your image builds or test any changes to the `Dockerfile`. Make sure you have Docker installed on your local machine and run the following command from the root of the project:
+
+```bash
+docker build --rm=false -t routemaster .
```

Comment:
It's not fixed though; mind updating it? If people start using `rm=false` as a default they'll soon come to us with "why my machine SLOW!" ... because they'll have 10s of containers using up resources in the background.